src/app.scss

```diff
@@ -0,0 +1,8 @@
+.main {
+  width: 100vw;
+  height: 100vh;
+
+  background-image: url('./assets/bg.jpeg');
```

Comment:
Where did this image come from? Is it royalty free?

Reply:
Yes, the image is royalty free, I downloaded it from pexels.com, a free stock photos and video platform